_rules/motion-replaced-5d9784.md

```diff
@@ -0,0 +1,193 @@
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility tree
+
+authors:
+  - João Vicente
+  - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML document](https://dom.spec.whatwg.org/#concept-document).
+
+## Expectation
+
+For each target element with [functionality](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html#dfn-functionality) that can be operated by device motion or user motion can also be operated by [user interface components](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html#dfn-user-interface-component)
+
+## Assumptions
+
+- This test assumes that motion is not [essential](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html#dfn-essential) for the function
+
+## Accessibility Support
+
+- Device orientation events may include sensitive data. Implementations must fire events only on [secure browsing contexts](https://www.w3.org/TR/secure-contexts/)
+
+## Background
+
+- [Understanding Success Criterion 2.5.4: Motion Actuation](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html)
+- [G213: Provide conventional controls and an application setting for motion activated input](https://www.w3.org/WAI/WCAG21/Techniques/general/G213.html)
+- [DeviceOrientation Event Specification](https://www.w3.org/TR/orientation-event/)
+- [Detecting device orientation](https://developer.mozilla.org/en-US/docs/Web/API/Detecting_device_orientation)
+- [DeviceMotionEvent](https://developer.mozilla.org/en-US/docs/Web/API/DeviceMotionEvent)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+[HTML document](https://dom.spec.whatwg.org/#concept-document) does not have [functionality](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html#dfn-functionality) that can be operated by device motion or user motion
+
+```html
+<div>document content</div>
+```
+
+#### Passed Example 2
+
+[HTML document](https://dom.spec.whatwg.org/#concept-document) has [functionality](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html#dfn-functionality) that can be operated by device motion or user motion and can also be operated by [user interface components](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html#dfn-user-interface-component)
+
+```html
+<html>
+  <head>
+    <script>
+      function load() {
+        //slider behavior
+        var slider = document.getElementById("motionSlider");
+        var output = document.getElementById("demo");
+        output.innerHTML = slider.value;
+        slider.oninput = function () {
+          output.innerHTML = this.value;
+        }
+
+        document.getElementById('increase').addEventListener('click', function() {

```

Comment:
I think I'd rather go for a named Javascript function (out of `load`) and a `onClick` on the button. Makes it easier to understand what happen since it is immediate that clicking on the button does something.